photon-all/build.gradle

```diff
@@ -16,9 +16,7 @@ apply plugin: 'scala'
 apply plugin: 'com.github.johnrengelman.shadow' // See doc at: http://imperceptiblethoughts.com/shadow/.
 
 dependencies {
-  compile project(":photon-lib$scalaSuffix")
-  compile project(":photon-api$scalaSuffix")
-  compile project(":photon-client$scalaSuffix")
+  compile project(":photon-ml$scalaSuffix")
```

Comment:
What's the purpose of this change?

Reply:
It was generated by Intelij. Will remove